<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stretch</class>
 <widget class="QWidget" name="Stretch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblSample">
       <property name="text">
        <string>Sample: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="MantidQt::MantidWidgets::DataSelector" name="dsSample">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="loadLabelText" stdset="0">
        <string>Plot</string>
       </property>
       <property name="workspaceSuffixes" stdset="0">
        <stringlist>
         <string>_red</string>
         <string>_sqw</string>
        </stringlist>
       </property>
       <property name="fileBrowserSuffixes" stdset="0">
        <stringlist>
         <string>_red.nxs</string>
         <string>_sqw.nxs</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblRes">
       <property name="text">
        <string>Resolution: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="MantidQt::MantidWidgets::DataSelector" name="dsResolution">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoLoad">
        <bool>true</bool>
       </property>
       <property name="workspaceSuffixes" stdset="0">
        <stringlist>
         <string>_res</string>
        </stringlist>
       </property>
       <property name="fileBrowserSuffixes" stdset="0">
        <stringlist>
         <string>_res.nxs</string>
        </stringlist>
       </property>
       <property name="showLoad">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOptions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Fit Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="lblBackground">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Background: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cbBackground">
          <item>
           <property name="text">
            <string>Sloping</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Flat</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Zero</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QCheckBox" name="chkSequentialFit">
          <property name="text">
           <string>Sequential Fit</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="chkElasticPeak">
          <property name="text">
           <string>Elastic Peak</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="treeSpace"/>
     </item>
     <item>
      <widget class="MantidQt::MantidWidgets::PreviewPlot" name="ppPlot" native="true">
       <property name="canvasColour" stdset="0">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
       <property name="showLegend" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_44">
     <item>
      <widget class="QGroupBox" name="gbOutput">
       <property name="title">
        <string>Output Options</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblPlotResult">
          <property name="text">
           <string>Plot Result: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbPlot">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sigma</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Beta</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_18">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="chkSave">
          <property name="text">
           <string>Save Result</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::DataSelector</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/DataSelector.h</header>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::PreviewPlot</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/PreviewPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
